ios/Trifle/Sources/TrifleCertificate.swift

```diff
@@ -6,11 +6,36 @@
 import Foundation
 import Wire
 
-public class TrifleCertificate : Codable {
+public class TrifleCertificate {
```

Comment:
why not have the class conform to `Equatable` directly instead of using an extension? Comment applies to all the other Trifle* containers